.github/workflows/experiment.yml

```diff
@@ -206,3 +206,4 @@ jobs:
         run: gh workflow run web.yml --ref "${{ github.ref }}"
         env:
           GH_TOKEN: ${{ github.token }}
+          REPORT_NAME: ${{ env.REPORT_NAME }}
```

Comment:
have you actually verify that this works? because `gh workflow run` does a `workflow dispatch` and it's really strange to me that this reads environment variables

I think what you actually need to is to either:
1. define a dedicated, **optional** input to the `web.yml` workflow and pass report name that way. then send the notification only if that input has a value
2. do not pass any information that the notification should be sent and instead within web.yml detect whether the commit that we're publishing is one that adds a new report. this should be surprisingly easy because we control commit messages

variant 2 has a drawback that it will send duplicate notifications on workflow reruns on the same commit.